src/stylesheets/components/_card.scss

```diff
@@ -1,27 +1,52 @@
-@mixin card-equal-height {
-  @include at-media("tablet") {
-    @include u-display("flex");
-    @include u-flex("column");
-    @include u-height("full");
+/* TODO:
+ * Add a way to change the flag breakpoint as a modifier/class?
+ * Is card grid gap be different from other gaps?
+ * Fix flag and set-aspect
+ */
 
-    .usa-card__body {
-      @include u-flex("fill");
-      flex-basis: auto; // IE 11
```

Comment:
Removing this causes issues in IE11.

![image](https://user-images.githubusercontent.com/3385219/80504054-4d85a700-8938-11ea-8011-9ed541a168ca.png)

![image](https://user-images.githubusercontent.com/3385219/80504213-773ece00-8938-11ea-8021-8438e2062cec.png)